anax-in-container/horizon-container

```diff
@@ -181,6 +184,13 @@ start() {
 		checkrc $? "docker run"
 	fi
 
+
+	if [[ -n $HZN_ICP_CA_CERT_PATH ]]; then
+		docker cp $HZN_ICP_CA_CERT_PATH $DOCKER_NAME:/usr/local/share/ca-certificates/icp.crt
+		docker exec $DOCKER_NAME update-ca-certificates
+		docker exec $DOCKER_NAME /root/anax.service restart
+	fi
+
```

Comment:
i'm concerned about this approach, because if `docker run` returns before the container is fully up, the `docker exec` could fail. I think the better way to handle this is: if `$HZN_ICP_CA_CERT_PATH` is set then add another mount for it to the `docker run`, and then have `anax.service` (which runs inside the container) detect the existence of that file/dir, copy it to `/usr/local/share/ca-certificates/` and do `update-ca-certificates`. If it does this at the right time, it won't even require a 2nd restart of anax (because `anax.service` already modifies anax.json and restarts anax).

Reply:
Also, `anax.json` has a field `Edge.CACertsPath`. How does that interact with the new variable in `/etc/default/horizon` ?

Reply:
They do not interact directly, certificates in the CACertsPath will still be trusted.